modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -39,6 +39,107 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-176607. Verify that is not possible to edit an organization information without EVP Manager role"
```

Comment:
Hey @heiditrindade. I`m trying a new approach in the test descriptions text:
what do you think about this above suggestion:
LPS-176607 - Financial and User roles should not be able to edit the organization."

Reply:
Ok, I will do as you suggested.

Reply:
@heiditrindade, Julio mentioned the aligh with Nikki to set te commit title aways referring to the US Ticket. and internally in tasks referring the current automation ticket. could you change your commit lps number, please?

Reply:
I'm not following... The ticket in the description is the ticket for the automation. The ticket in the title is the story ticket. It is as you said in the comment. Am I missing something? If I am, please let me know.

Reply:
sorry. my bad